src/.nuget/dir.props

```diff
@@ -111,48 +115,22 @@
   </Choose>
 
   <ItemGroup Condition="$(SupportedPackageOSGroups.Contains(';Linux;'))">
-    <OfficialBuildRID Include="alpine.3.4.3-x64" />
-    <OfficialBuildRID Include="debian.8-x64" />
-    <OfficialBuildRID Include="fedora.23-x64" />
-    <OfficialBuildRID Include="fedora.24-x64" />
     <OfficialBuildRID Include="linux-x64" />
     <OfficialBuildRID Include="linux-arm">
       <Platform>arm</Platform>
     </OfficialBuildRID>
-    <OfficialBuildRID Include="opensuse.42.1-x64" />
-    <OfficialBuildRID Include="rhel.7-x64" />
     <OfficialBuildRID Include="tizen.4.0.0-armel">
```

Comment:
I think it is probably worth putting a comment on this RID similar to what we did in core-setup and corefx.